app/assets/javascripts/application.js.erb

```diff
@@ -42,8 +42,9 @@
 //= require select2_customization
 //= require shared/inline_editing
 //= require activestorage
-//= require turbolinks
 //= require marvinjslauncher
+//= require_directory ./repositories/renders
```

Comment:
isn't it working without './'?

and I think 'renderers' is more correct name

Reply:
Fixed